Plandomizer.py

```diff
@@ -714,8 +714,11 @@ def fill(self, window, worlds, location_pools, item_pools):
         for (location_name, record) in self.pattern_dict_items(locations):
             if record.item is None:
                 continue
-
-            valid_items = self.get_valid_items_from_record(world.itempool, used_items, record)
+            valid_items = []
+            if record.item == "#Vanilla": # Get vanilla item at this location from the location table
+                valid_items.append(location_table[location_name][4]) 
```

Comment:
Suggestion for a future PR: turn location_table values into a namedtuple, so we don't have to remember we have a lookup to index 4 in the tuple.